consensus/src/chained_bft/epoch_manager.rs

```diff
@@ -295,9 +300,11 @@ impl<T: Payload> EpochManager<T> {
             self.config.max_block_size,
         );
 
+        info!("Create Pacemaker");
         let pacemaker =
             self.create_pacemaker(self.time_service.clone(), self.timeout_sender.clone());
 
+        info!("Create ProposerElection");
```

Comment:
this will show up in basically every log except those that only enable error and warn. is that the intent?

Reply:
yeah, this is only for epoch transition which should be rare enough